src/components/InventoryHostStaleness/BaseDropDown.js

```diff
@@ -1,7 +1,14 @@
 import React, { useState } from 'react';
 import PropTypes from 'prop-types';
-import { Button, Flex, FlexItem, Popover } from '@patternfly/react-core';
-import { Select, SelectOption } from '@patternfly/react-core/deprecated';
+import {
+  Button,
+  Flex,
+  FlexItem,
+  MenuToggle,
+  Popover,
+  Select,
+  SelectOption,
+} from '@patternfly/react-core';
```

Comment:
Let's try to revert this. While we do get a component NOT from `'@patternfly/react-core/deprecated'` it may not be the same and behave differently. 

Reverting to the old imports will probably also not require any changes below.